core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -205,7 +231,14 @@ private[spark] class EventLoggingListener(
   override def onBlockUpdated(event: SparkListenerBlockUpdated): Unit = {}
 
   // No-op because logging every update would be overkill
-  override def onExecutorMetricsUpdate(event: SparkListenerExecutorMetricsUpdate): Unit = { }
+  override def onExecutorMetricsUpdate(event: SparkListenerExecutorMetricsUpdate): Unit = {
+    // In order to avoid the logged event consumes too much storage size, taskMetrics would not
+    // be logged into event log file currently
+    val lightEvent = SparkListenerExecutorMetricsUpdate(
+      event.execId, event.executorMetrics, Seq.empty)
+    executorIdToLatestMetrics.update(lightEvent.execId, lightEvent)
```

Comment:
nit: more idiomatic scala to write `map.update(key,value)` with `map(key) = value` (scala syntax sugar for `update` method)